src/main/java/Task.java

```diff
@@ -0,0 +1,45 @@
+public class Task {
+    protected String description;
```

Comment:
While protected may appear sufficient, these attributes can still be accessible by other functions once you start putting them into packages. Consider changing them to `private`